index.html

```diff
@@ -1131,6 +1131,11 @@ <h2>Class Definitions</h2>
             </span>
         </p>
 
+        <p>
+            Please note that all vocabulary terms and values are case sensitive.
+            This is also true for the serialization of the information model (Section <a href="#sec-td-serialization"></a>).
```

Comment:
I am just wondering. You are referring to "information model" and linking to the representation format. Is this intended?

Reply:
I mean... either the text is wrong or the link is wrong